docs/src/modules/components/AppTableOfContents.js

```diff
@@ -36,16 +36,17 @@ const useStyles = makeStyles((theme) => ({
   },
   item: {
     fontSize: 13,
-    padding: theme.spacing(0.5, 0, 0.5, 1),
-    borderLeft: '4px solid transparent',
-    boxSizing: 'content-box',
+    // paddingLeft: theme.spacing(1) where `border` is used as the "active" marker
+    paddingLeft: theme.spacing(0.5),
+    borderLeft: `${theme.spacing(0.5)}px solid transparent`,
+    boxSizing: 'border-box',
     '&:hover': {
-      borderLeft: `4px solid ${
+      borderLeft: `${theme.spacing(0.5)}px solid ${
```

Comment:
I doubt the spacing helper is related to the thickness of the border

Reply:
makes sense

Reply:
done